test/com/facebook/buck/apple/testdata/prebuilt_apple_framework_links/app/main.m

```diff
@@ -0,0 +1,10 @@
+#import <BuckTest/BuckTest.h>
```

Comment:
@robbertvanginkel Just out of curiousity, I'm wondering if @import works, since you already have the `.modulemap` file.  That would be a good thing to support if it doesn't already.